python/instrumentation/openinference-instrumentation-portkey/src/openinference/instrumentation/portkey/_request_attributes_extractor.py

```diff
@@ -53,6 +53,12 @@ def get_extra_attributes_from_request(
 
         yield SpanAttributes.LLM_INVOCATION_PARAMETERS, safe_json_dumps(invocation_params)
 
+        if prompt_id := invocation_params.get("prompt_id"):
+            yield SpanAttributes.PROMPT_ID, prompt_id
+
```

Comment:
it looks like you can produce a prompt URL pretty easily if you can: https://app.portkey.ai/prompts/ecc1de52-83dc-4846-a963-8d69f86dfee3 for example